fendermint/actors/chainmetadata/src/shared.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::{
+    multihash::{Code, MultihashDigest},
+    Cid,
+};
+use fil_actors_runtime::Array;
```

Comment:
Why do we need this `Array` type, is it an alias? Can we use the `fvm_ipld_amt` crate to not need this runtime?

Reply:
IIRC it provides some helper methods, but we shouldn't need those that badly.

Reply:
its an alias, should be easy to replace